### ISSUE TITLE
Color Contrast (via accessibility testing)

### ISSUE THREAD

**Issue by rebeccacremona** [harvard-lil/capstone#505]
These screenshots, which are big, have a greater apparent contrast than the real thing.

Landing page:

![image](https://user-images.githubusercontent.com/11020492/45900672-73d38000-bdae-11e8-8cec-2c183b860ff4.png)
Orange on white
Contrast ratio: 2.91 
AA level (4.50): #dc3b00/#ffffff 

Gray on white
Contrast ratio: 2.85 
AA level (4.50): #747494/#ffffff 

![image](https://user-images.githubusercontent.com/11020492/45901219-34a62e80-bdb0-11e8-8173-86c9980552c4.png)
blue (hover/focus) on black
Contrast ratio: 3.18 
AA level (4.53): #6592ff/#2f2f2f 

![image](https://user-images.githubusercontent.com/11020492/45901829-556f8380-bdb2-11e8-8cdd-5cd3541b64b5.png)
orange (hover/focus) on purple
Contrast ratio: 3.54 
AA level (4.52): #ffefcf/#6350fd 
(the other footer links are fine, 4.65)

![image](https://user-images.githubusercontent.com/11020492/45902055-3de4ca80-bdb3-11e8-96a7-f2140bc6ab4a.png)
blue (hover) on white
Contrast ratio: 4.21 
AA level (4.53): #ffffff/#0070f5 
very close!

**Comment by rebeccacremona** [comment]
About page

![image](https://user-images.githubusercontent.com/11020492/45902321-1fcb9a00-bdb4-11e8-9873-5845681711bc.png)
blue on white (hover, focus)
Contrast ratio: 4.21 
AA level (4.53): #0070f5/#ffffff 
(applies to in-text links on all pages, not just About)

![image](https://user-images.githubusercontent.com/11020492/45902638-11ca4900-bdb5-11e8-88b7-12e772afb0c3.png)
blue (hover/focus) on orange
Contrast ratio: 2.90 
AA level (4.54): #0058c4/#ffd21b 

Return to top link changes to the same color as the background on hover/focus
![image](https://user-images.githubusercontent.com/11020492/45902696-4dfda980-bdb5-11e8-9bda-016955ac5197.png)

**Comment by rebeccacremona** [comment]
Tools > Overview, Tools > Bulk Data, Login/Registration

![image](https://user-images.githubusercontent.com/11020492/45902958-2529e400-bdb6-11e8-8804-4472bb96bb7e.png)
Blue on black (or blue on black during hover/focus)
Contrast ratio: 3.18 
AA level (4.53): #6592ff/#2f2f2f

**Comment by rebeccacremona** [comment]
Tools > API
![image](https://user-images.githubusercontent.com/11020492/45903146-aed9b180-bdb6-11e8-80a8-b7f334756be2.png)
blue on blue
Contrast ratio: 3.15 
AA level (4.51): #0063cf/#d2e7ff

purple on blue
Contrast ratio: 4.06 
AA level (4.50): #5b45fb/#d2e7ff 

red on blue
Contrast ratio: 3.58 
AA level (4.32): #d5001c/#d2e7ff

![image](https://user-images.githubusercontent.com/11020492/45903322-450dd780-bdb7-11e8-9c8e-efeac4947890.png)
the footer is black on black :-) 
orange hover return-to-top is fine, see above for blue-on-black hover of nav

**Comment by rebeccacremona** [comment]
Login/registration footer "return to top" has no hover/focus style

**Comment by rebeccacremona** [comment]
Contact page

![image](https://user-images.githubusercontent.com/11020492/45903969-80a9a100-bdb9-11e8-813e-b42b64a656a5.png)
purple on yellow
Contrast ratio: 3.54 
AA level (4.52): #5035f8/#ffd21b 

blue (hover/focus) on yellow
Contrast ratio: 2.90 
AA level (4.54): #0058c4/#ffd21b 

![image](https://user-images.githubusercontent.com/11020492/45904354-c74bcb00-bdba-11e8-8258-acd2e358c7fb.png)

yellow on white (on hover/focus)
Contrast ratio: 1.45 
AA level (4.52): #8e7400/#ffffff
(recommendation from the software is terrible here)

**Comment by rebeccacremona** [comment]
Gallery > Overview

![image](https://user-images.githubusercontent.com/11020492/45904461-1eea3680-bdbb-11e8-9022-80cf51fb5d79.png)
purple on black
Contrast ratio: 2.61 
AA level (4.52): #9087ff/#2f2f2f 

![image](https://user-images.githubusercontent.com/11020492/45904690-c8c9c300-bdbb-11e8-914b-dfcae40a292c.png)
blue on red (hover/focus)
Contrast ratio: 1.45 
AA level (4.54): #002d6c/#ff654d

![image](https://user-images.githubusercontent.com/11020492/45904767-fdd61580-bdbb-11e8-8d13-290fc72a4bae.png)
Orange on red
Contrast ratio: 2.01 
AA level (4.53): #ffd21b/#af2d00

**Comment by rebeccacremona** [comment]
Wordclouds years have hover but not focus styles

**Comment by anastasia** [comment]
Done!
